### ISSUE TITLE
Tutorial to use the program

### ISSUE THREAD

**Issue by dlare** [Ar1i/PokemonGo-Bot#4067]
######################## RULES. READ BEFORE POSTING! ########################

-> DONT OPEN A NEW ISSUE IF YOU DIDNT SEARCH YOUR PROBLEM IN THE ISSUES!

-> DONT OPEN A NEW ISSUE IF YOUR ISSUE GOT ANSWERED PERVIOUSLY!

-> DONT OPEN A NEW ISSUE IF YOUR "ISSUE" IS A QUESTION/SUGGESTION!

-> DONT OPEN A NEW ISSUE IF YOU DONT USE THE NEWEST VERSION!

If you do not respect them, your issue gets closed without any comment

#########################################################################

Describe your problem!

Add a screenshot!

Choose a good title!

Dont forget to delete this template!

Post.

**Comment by shadowahs** [comment]
Download WinRAR (if you don't already have it) from here: http://www.rarlab.com/
   -Select language and then 32 Bit

Finish Installing WinRAR, then go to this link: https://github.com/Ar1i/PokemonGo-Bot/blob/master/Builds-Only/PokemonGo.RocketAPI.Console.exe
   -Hit Download and while it's downloading, make a new folder on your desktop.

Once the download is finished, drag/copy the download into the new folder and select the application you just moved and start it. 

Configure the bot the way you want and start it up

**Comment by shadowahs** [comment]
For future reference, only post Issues the bot is having on the issues tab, but i had the same problem as you in the beginning so I'd like to help.
